javascript/javascript2/week1/hyfbay/main.js

```diff
@@ -0,0 +1,17 @@
+console.log('Script loaded');
+let testProductsNames=["flatscreen", "phone", "radio", "fridge"];
+
+
+
+let ul = document.getElementById("productsUl");
+
+
+function renderProducts(products){
+ const productList = products.map((productName)=>{
+const li = document.createElement("li");
+li.innerHTML = productName;
+ul.appendChild(li);
+  });
+}
+console.log(renderProducts(testProductsNames));
+console.log(getAvailableProducts());
```

Comment:
This is only half of the exercise! You should only use `testProductsNames` for a quick prototype. But your page should actually show the products that are returned by the `getAvailableProducts()` function!!

`let testProductsNames = getAvailableProducts();`